bus-mapping/src/circuit_input_builder/tracer_tests.rs

```diff
@@ -246,25 +246,75 @@ fn tracer_err_insufficient_balance() {
     let next_step = block.geth_traces[0].struct_logs.get(index + 1);
     assert_eq!(step.error, None);
     assert_eq!(next_step.unwrap().op, OpcodeId::PUSH2);
-    assert_eq!(next_step.unwrap().stack, Stack(vec![Word::zero()])); // success = 0
+    assert_eq!(next_step.unwrap().stack, Stack(vec![Word::zero()])); // failure = 0
 
     let mut builder = CircuitInputBuilderTx::new(&block, step);
-    builder.builder.sdb.set_account(
-        &ADDR_A,
-        Account {
-            nonce: Word::zero(),
-            balance: Word::from(555u64), /* same value as in
-                                          * `mock::new_tracer_account` */
-            storage: HashMap::new(),
-            code_hash: Hash::zero(),
-        },
-    );
     assert_eq!(
         builder.state_ref().get_step_err(step, next_step).unwrap(),
         Some(ExecError::InsufficientBalance)
     );
 }
 
+#[test]
+fn tracer_call_success() {
+    let code_a = bytecode! {
+    PUSH1(0x0) // retLength
+    PUSH1(0x0) // retOffset
+    PUSH1(0x0) // argsLength
+    PUSH1(0x0) // argsOffset
+    PUSH32(Word::from(0x1000)) // value
+    PUSH32(Word::from(0x000000000000000000000000000000000cafe001)) // addr
+    PUSH32(0x1_0000) // gas
+    CALL
+
+    PUSH2(0xaa)
+    };
+    let code_b = bytecode! {
+    STOP
+    };
```

Comment:
```suggestion
    let code_a = bytecode! {
        PUSH1(0x0) // retLength
        PUSH1(0x0) // retOffset
        PUSH1(0x0) // argsLength
        PUSH1(0x0) // argsOffset
        PUSH32(Word::from(0x1000)) // value
        PUSH32(Word::from(0x000000000000000000000000000000000cafe001)) // addr
        PUSH32(0x1_0000) // gas
        CALL

        PUSH2(0xaa)
    };
    let code_b = bytecode! {
        STOP
    };
```

Reply:
updated in [bda71b7](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/630/commits/bda71b728670f0567cf5e8961af6590f7d6b372f). thanks